web-common/src/features/dashboards/filters/measure-filters/MeasureFilter.svelte

```diff
@@ -31,10 +36,13 @@
       active = false;
     }
   }
+
+  // TODO: in the next round of refactor update upstream to use MeasureFilterEntry
+  let filter: MeasureFilterEntry | undefined;
```

Comment:
Nit: Just my preference, but I generally like avoiding the secondary declaration unless it's for caching reasons or a prop. Ensures the types are being accurately inferred and saves a couple lines.

Reply:
The case where this is undefined is when the menu is opened for the 1st time. Same applies to other components as well.